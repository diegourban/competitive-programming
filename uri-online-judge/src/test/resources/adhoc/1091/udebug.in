3
1 1
2 1
1 5
-1000 -1000
5
-100 100
-1000 1003
1 9999
-999 -1001
5 -10
-100 -10000
2
0 0
100 122
-100 -9
4
600 0
1 1
-20 1
2222 2999
1 0
9
0 0
0 1
0 -1
1 0
-1 0
1 1
-1 -1
-1 1
1 -1
0 0
0