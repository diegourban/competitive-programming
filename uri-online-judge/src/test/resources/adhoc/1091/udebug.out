divisa
divisa
SO
NO
NE
SO
SE
divisa
NE
SO
NO
NO
NE
divisa
divisa
divisa
divisa
divisa
NE
SO
NO
SE
divisa
